charts/workflows/templates/hostpath-clusterpolicy.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: hostpath
+spec:
+  validationFailureAction: Enforce
+  rules:
+    - name: check-hostpath
+      match:
+        resources:
+          kinds:
+            - Pod
+          namespaceSelector:
+            matchLabels:
+              app.kubernetes.io/managed-by: sessionspaces
+      context:
+        - name: values
+          configMap:
+            name: sessionspaces
+            namespace: "{{ `{{request.object.metadata.namespace}}` }}"
+      validate:
+        message: This hostPath is not allowed
```

Could be more explicit about what the path should be - bit better for someone who experiences a denial
```suggestion
        message: The hostPath must point to the session directory
```